lambda/create_db_users.tf

```diff
@@ -27,7 +28,7 @@ resource "aws_lambda_function" "create_db_users_lambda_function" {
 }
 
 resource "aws_kms_ciphertext" "environment_vars_create_db_users" {
-  for_each  = local.count_create_db_users == 0 ? {} : { db_admin_user = var.db_admin_user, db_admin_password = var.db_admin_password, db_url = "jdbc:postgresql://${var.db_url}:5432/consignmentapi" }
+  for_each  = local.count_create_db_users == 0 ? {} : { db_admin_user = var.db_admin_user, db_admin_password = var.db_admin_password, db_url = "jdbc:postgresql://${var.db_url}:5432/consignmentapi", database_name = "consignmentapi" }
```

Comment:
Do we want to leave the db name hardcoded? Seems a little specific for the terraform modules?

Reply:
It is but because the lambdas are all self contained so this is only for the create db users lambda, it's ok I think. This value is used by the code to see which database to update so it's never going to change. I can put it in the locals if that's better, what do you think?